src/pages/costModels/components/rateForm/rateForm.tsx

```diff
@@ -106,14 +121,13 @@ const RateFormBase: React.FunctionComponent<RateFormProps> = ({
             style={style}
             id="metric"
             label={messages.Metric}
+            placeholderText="Select..."
             value={metric}
-            onChange={setMetric}
+            onChange={val => {
+              setMeasurement(undefined);
```

Comment:
I believe this is expected

`UPDATE_METRIC` sets the required error on measurement if it was dirty
<img width="371" alt="image" src="https://user-images.githubusercontent.com/5322142/160695382-189f1fbf-733a-4120-873c-8f2a5629e0c7.png">
this does mean that my manually resetting of measurement is redundant though, so i can remove that

tests also seem to indicate that this is expected behavior
<img width="545" alt="image" src="https://user-images.githubusercontent.com/5322142/160695601-68475161-8d7a-4492-8a6a-51ab8f419e81.png">

Reply:
Ideally, the state should be reset when a new metric is selected. However, we can create a new Jira issue for this, since it appears to be an existing bug. My thought was to clean this up while you were working in this area.

Reply:
It seems odd that this message would be displayed after we reset the menu. However, I'll look at creating a new Jira issue for this, since it appears to be existing behavior.